libs/langchain/tests/unit_tests/test_text_splitter.py

```diff
@@ -1091,27 +1092,22 @@ def test_md_header_text_splitter_preserve_headers_2() -> None:
     ]
     markdown_splitter = MarkdownHeaderTextSplitter(
         headers_to_split_on=headers_to_split_on,
-        strip_headers=False,
+        return_each_line=False,
     )
     output = markdown_splitter.split_text(markdown_document)
     expected_output = [
         Document(
-            page_content="# Foo  \n## Bar  \nHi this is Jim  \nHi this is Joe",
```

Comment:
let's leave all these tests as-is. I'm a little suspicious this PR has breaking changes because of these test changes.